api/swagger.yml

```diff
@@ -981,6 +981,26 @@ components:
         - gc_commits_location
         - gc_addresses_location
 
+    PrepareGCUncommittedRequest:
+      type: object
+      properties:
+        continuation_token:
+          type: string
+
+    PrepareGCUncommittedResponse:
+      type: object
+      properties:
+        run_id:
+          type: string
+        location:
```

Comment:
We might want to add the location of the copy object file list to this command as well.
Suggest to change the name to be more specific

Reply:
gc_uncommitted_location